crypto/property/property_parse.c

```diff
@@ -579,7 +576,11 @@ int ossl_property_parse_init(OPENSSL_CTX *ctx)
 
     /* Pre-populate the two Boolean values */
     if ((ossl_property_true = ossl_property_value(ctx, "yes", 1)) == 0
-        || (ossl_property_false = ossl_property_value(ctx, "no", 1)) == 0)
+        || (ossl_property_false = ossl_property_value(ctx, "no", 1)) == 0
+        /* Create values for the built-in providers */
+        || ossl_property_value(ctx, "default", 1) == 0
```

Comment:
These aren't required.  Values are created on demand when parsing definitions.

The reason that _yes_ and _no_ are treated specially is to avoid lots of look ups when checking Booleans.  With this change we're moving away from Booleans and we could reconsider having these special values entirely.

Reply:
I backed this change out.